platform/spark/modules/eGUI_screens/UI.cpp

```diff
@@ -9,6 +29,7 @@ extern "C" {
 #include "brewpi_boot_screen.h"
 
 uint8_t UI::init() {
+    eGuiSettings storedSettings;
```

Comment:
I would make this a global, since there's no easy way to free the flashee device. Also you may need other config items relating to eGUI later in the code.